### PR TITLE
Change badge on request form

### LINKED ISSUE #3044: For items that are in resource sharing change the status in the Request form to be Not Available
For items that are in resource sharing change the status in the Request form to be `Not Available`.
Remove the `- Resource sharing Request`.

Screen Shot 2022-06-15 at 1 01 59 PM

### PR DESCRIPTION
**Pull request by maxkadel**
- Remove "Resource Sharing" from badge

Closes #3044

### CODE REVIEW

File: app/models/requests/requestable_decorator.rb [hunk 1]
```diff
@@ -123,14 +123,18 @@ def status_badge
                   else
                     "badge-danger"
                   end
-      status = if requestable.status_label.nil? || requestable.status == requestable.status_label
-                 requestable.status
-               else
-                 requestable.status + ' - ' + requestable.status_label
-               end
+      status = status_for_badge
```
Comment (sandbergja) [inline app/models/requests/requestable_decorator.rb:126]: Is this assignment needed? (couldn't we just call status_for_badge on the next line?)

File: app/models/requests/requestable_decorator.rb [hunk 2]
```diff
@@ -123,14 +123,18 @@ def status_badge
...
       content_tag(:span, status, class: "availability--label badge #{css_class}")
     end
 
+    def status_for_badge
```
Comment (sandbergja) [inline app/models/requests/requestable_decorator.rb:130]: Thanks for refactoring this out into its own method!

**Review by sandbergja — CHANGES_REQUESTED** [review]

File: app/models/requests/requestable_decorator.rb (hunk 1 above)
Comment (maxkadel) [reply app/models/requests/requestable_decorator.rb:126]: So true. I feel silly.

**Review by sandbergja — APPROVED** [review]

**Comment by sandbergja** [comment]
Thanks @maxkadel

**Pull request merged**